installer/PowerToysSetup/Product.wxs

```diff
@@ -254,6 +254,75 @@
+          <RegistryValue Type="string" Key="InprocServer32\1.0.0.0" Name="CodeBase" Value="file:///[#Svg_PreviewHandler_Dll]" />
+        </RegistryKey>
+        <!-- Registry Key for Class Registration of Markdown Preview Handler -->
+        <RegistryKey Root="HKCU" Key="Software\Classes\CLSID\{45769bcc-e8fd-42d0-947e-02beef77a1f5}">
+          <RegistryValue Type="string" Value="MarkdownPreviewHandler.MarkdownPreviewHandler" />
+          <RegistryValue Type="string" Name="DisplayName" Value="Markdown Preview Handler" />
+          <RegistryValue Type="string" Name="AppID" Value="{CF142243-F059-45AF-8842-DBBE9783DB14}" />
+          <RegistryValue Type="string" Key="Implemented Categories\{62C8FE65-4EBB-45e7-B440-6E39B2CDBF29}" Value="" />
+          <RegistryValue Type="string" Key="InprocServer32" Value="mscoree.dll" />
+          <RegistryValue Type="string" Key="InprocServer32" Name="Assembly" Value="MarkdownPreviewHandler, Version=1.0.0.0, Culture=neutral, PublicKeyToken=201d23296ab86a5a" />
+          <RegistryValue Type="string" Key="InprocServer32" Name="Class" Value="MarkdownPreviewHandler.MarkdownPreviewHandler" />
+          <RegistryValue Type="string" Key="InprocServer32" Name="RuntimeVersion" Value="v4.0.30319" />
+          <RegistryValue Type="string" Key="InprocServer32" Name="ThreadingModel" Value="Both" />
+          <RegistryValue Type="string" Key="InprocServer32" Name="CodeBase" Value="file:///[#Md_PreviewHandler_Dll]" />
+          <RegistryValue Type="string" Key="InprocServer32\1.0.0.0" Name="Assembly" Value="MarkdownPreviewHandler, Version=1.0.0.0, Culture=neutral, PublicKeyToken=201d23296ab86a5a" />
+          <RegistryValue Type="string" Key="InprocServer32\1.0.0.0" Name="Class" Value="MarkdownPreviewHandler.MarkdownPreviewHandler" />
+          <RegistryValue Type="string" Key="InprocServer32\1.0.0.0" Name="RuntimeVersion" Value="v4.0.30319" />
+          <RegistryValue Type="string" Key="InprocServer32\1.0.0.0" Name="CodeBase" Value="file:///[#Md_PreviewHandler_Dll]" />
+        </RegistryKey>
+        <!-- Registry Key for AppID registration -->
+        <RegistryKey Root="HKCU" Key="Software\Classes\AppID\{CF142243-F059-45AF-8842-DBBE9783DB14}">
+          <RegistryValue Type="expandable" Name="DllSurrogate" Value="%SystemRoot%\system32\prevhost.exe" />
+        </RegistryKey>
+        <!-- Add Svg preview handler to preview handlers list -->
+        <RegistryKey Root="HKCU" Key="SOFTWARE\Microsoft\Windows\CurrentVersion\PreviewHandlers">
+          <RegistryValue Type="string" Name="{ddee2b8a-6807-48a6-bb20-2338174ff779}" Value="Svg Preview Handler" />
+        </RegistryKey>
+        <!-- Add Markdown preview handler to preview handlers list -->
+        <RegistryKey Root="HKCU" Key="SOFTWARE\Microsoft\Windows\CurrentVersion\PreviewHandlers">
+          <RegistryValue Type="string" Name="{45769bcc-e8fd-42d0-947e-02beef77a1f5}" Value="Markdown Preview Handler" />
+        </RegistryKey>
```

Comment:
[Nit] Software here in Caps. Better to keep it consistent in Camel case as "Software" across all the entries.

Reply:
I will prefer to update all of them to upper case `SOFTWARE` as it's the casing in registry.

Reply:
It seems to be `Software` in HKCU and `SOFTWARE` in HKLM.

Reply:
Updated all entries to `Software`